plexapi/library.py

```diff
@@ -2033,6 +2040,31 @@ def sync(self, bitrate, limit=None, **kwargs):
         kwargs['policy'] = Policy.create(limit)
         return super(MusicSection, self).sync(**kwargs)
 
+    def sonicAdventure(
+            self,
+            start: Track | int,
+            end: Track | int,
+            **kwargs: Any,
+    ) -> list[Track]:
+        """ Returns a list of tracks from this library section that are part of a sonic adventure.
+        ID's should be of a track, other ID's will return an empty list or items itself or an error.
+
+                Parameters:
+                    start (Track | int): The :class:`~plexapi.audio.Track` or ID of the first track in the sonic adventure.
+                    end (Track | int): The :class:`~plexapi.audio.Track` or ID of the last track in the sonic adventure.
+                    kwargs: Additional parameters to pass to :func:`~plexapi.base.PlexObject.fetchItems`.
+
+                Returns:
+                    List[:class:`~plexapi.audio.Track`]: a list of tracks from this library section
+                    that are part of a sonic adventure.
+        """
```

Comment:
Documentation is rendering incorrectly.

```suggestion
        """ Returns a list of tracks from this library section that are part of a sonic adventure.
            ID's should be of a track, other ID's will return an empty list or items itself or an error.

            Parameters:
                start (Track | int): The :class:`~plexapi.audio.Track` or ID of the first track in the sonic adventure.
                end (Track | int): The :class:`~plexapi.audio.Track` or ID of the last track in the sonic adventure.
                kwargs: Additional parameters to pass to :func:`~plexapi.base.PlexObject.fetchItems`.

            Returns:
                List[:class:`~plexapi.audio.Track`]: a list of tracks from this library section
                that are part of a sonic adventure.
        """
```